tests/test_catalyst.py

```diff
@@ -14,38 +12,46 @@
 # pylint: disable=redefined-outer-name, unused-argument
 
 
-@pytest.fixture(scope="session")
-def loaders(tmp_path_factory):
-    path = tmp_path_factory.mktemp("catalyst_mnist")
-    train_data = MNIST(path, train=True, download=True)
-    valid_data = MNIST(path, train=False, download=True)
-    return {
-        "train": DataLoader(train_data, batch_size=32),
-        "valid": DataLoader(valid_data, batch_size=32),
-    }
-
-
 @pytest.fixture
 def runner():
     return dl.SupervisedRunner(
-        engine=get_available_engine(),
+        engine=catalyst.utils.torch.get_available_engine(cpu=True),
```

Comment:
Without `cpu=True`, it was hanging for a while for me, possibly initializing GPU/CUDA.